Formula/corsixth.rb

```diff
@@ -73,10 +66,10 @@ def install
 
   test do
     # Make sure I point to the right version!
-    lua = Formula["lua@5.3"]
+    lua = Formula["lua"]
 
```

Comment:
We can remove these lines since they're not used.

Reply:
yeah, just simplified the build a bit.